src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents the association between Person and Tag.
```

Comment:
Good but can be more specific. e.g. `Represents an adding or removing of a Tag to/from a Person.`